.gitignore

```diff
@@ -2,6 +2,7 @@ Makefile.config
 perl/Makefile.config
 
 # /
+/build
```

Comment:
Meson puts a .gitignore/.hgignore in the root of build directories. This should be unnecessary.

Reply:
is that new? if so this original PR was built before it did that

Reply:
Go figure, it is fairly new.

https://mesonbuild.com/FAQ.html#how-do-i-ignore-the-build-directory-in-my-vcs

Reply:
does muon also do this? is it a standard feature of the meson DSL?

Reply:
It's not part of the DSL.

It is just a really good idea to do it for generated directories that should never be checked into git. :D 

(Yes, muon does this as well.)

Reply:
I pulled the latest muon and it doesn't seem to be the case. @annacrombie thoughts?

Reply:
https://git.sr.ht/~lattis/muon/commit/975e988263c30308e6939a56bbb1026aaab3daa0

Reply:
I am an idiot and didn't pass `-a` to `ls`...